parsers/IL.py

```diff
@@ -55,14 +55,29 @@ def flatten_list(_2d_list) -> list:
     return flatten_list(cleaned_list)
 
 
+def fetch_price(zone_key="IL", session=None, target_datetime=None, logger=None) -> dict:
+    """Fetch price from IEC table."""
+    response = get(IEC_PRICE)
+    soup = BeautifulSoup(response.content, "lxml")
+
+    price = soup.find("td", class_="ms-rteTableEvenCol-6")
+
+    return {
+        "zoneKey": zone_key,
+        "currency": "NIS",
+        "datetime": arrow.now(TZ).datetime,
```

Comment:
Is it possible to retrieve the date at which the price was set?
This would make the underlying data more robust.

Reply:
No asthe price determined yearly, and not by "date"

Reply:
I see. Would it be possible to extract the "valid from" date?
That would be the most correct way to model the data

![image](https://user-images.githubusercontent.com/1655848/116536601-6f1bc480-a8e5-11eb-974c-fe7d6c32537f.png)

Reply:
Probably, and insert it in place of the arrow? How do you suggest implementing it?

Reply:
You can use `arrow.get` to parse the date. See more in their documentation: https://arrow.readthedocs.io/en/latest/

Reply:
Done